src/validation/userValidation.js

```diff
@@ -8,7 +8,7 @@ const loginValidation = [
     .withMessage(`email ${loginErrors.undefinedEmail}`)
     .isEmail()
     .withMessage(`email ${loginErrors.invalidEmail}`)
-    .matches(/@andela.com$/)
+    // .matches(/@andela.com$/)
```

Comment:
Is there a reason we are taking out the Andela email validation?. If there is, you can remove the line completely instead of commenting it out.